de.metas.serviceprovider/src/main/java/de/metas/serviceprovider/github/GithubImportProcess.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * #%L
+ * de.metas.serviceprovider.base
+ * %%
+ * Copyright (C) 2019 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.serviceprovider.github;
+
+import com.google.common.collect.ImmutableList;
+import de.metas.process.JavaProcess;
+import de.metas.serviceprovider.external.project.ExternalProjectReference;
+import de.metas.serviceprovider.external.project.ExternalProjectReferenceId;
+import de.metas.serviceprovider.external.project.ExternalProjectRepository;
+import de.metas.serviceprovider.importer.IssueImporterService;
+import de.metas.serviceprovider.importer.info.ImportIssuesRequest;
+import de.metas.util.Check;
+import de.metas.util.Services;
+import org.adempiere.service.ISysConfigBL;
+import org.compiere.SpringContextHolder;
+
+import static de.metas.serviceprovider.external.project.ExternalSystem.GITHUB;
+import static de.metas.serviceprovider.github.GithubImporterConstants.GitHubImporterSysConfig.ACCESS_TOKEN;
+
+public class GithubImportProcess extends JavaProcess
+{
+	private final String PARAM_NAME_EXTERNAL_PROJECT_REFERENCE_ID = "S_ExternalProjectReference_ID";
+	private final String PARAM_NAME_ISSUE_NO = "IssueNumbers";
+
+	private int externalProjectReferenceId;
+	private ImmutableList<String> issueNoList;
+
+	private final ExternalProjectRepository externalProjectRepository = SpringContextHolder.instance.getBean(ExternalProjectRepository.class);
+	private final IssueImporterService issueImporterService = SpringContextHolder.instance.getBean(IssueImporterService.class);
+	private final GithubImporterService githubImporterService = SpringContextHolder.instance.getBean(GithubImporterService.class);
+	private final ISysConfigBL sysConfigBL = Services.get(ISysConfigBL.class);
+
+
+	@Override protected String doIt() throws Exception
+	{
+		getParameters().forEach(param -> {
```

Comment:
check the annotation `de.metas.process.Param` and how it's used in other processes